package.json

```diff
@@ -494,6 +506,10 @@
         {
           "id": "SonarLint.ConnectedMode",
           "name": "SonarLint Connected Mode"
+        },
+        {
+          "id": "SonarLint.SecurityHotspots",
+          "name": "SonarLint Security Hotspots"
```

Comment:
At the moment the `SecurityHotspots` view is in the `SonarLint` view container. It can be easily moved to the panel , next to the problems view if needed.